x-pack/plugins/lens/public/indexpattern_datasource/indexpattern.tsx

```diff
@@ -663,6 +665,56 @@ export function getIndexPatternDatasource({
         ),
       ];
     },
+    getDeprecationMessages: () => {
+      const deprecatedMessages: React.ReactNode[] = [];
+      const useFieldExistenceSamplingKey = 'lens:useFieldExistenceSampling';
+      const isUsingSampling = core.uiSettings.get(useFieldExistenceSamplingKey);
+
+      if (isUsingSampling) {
+        const link = (
+          <EuiLink
+            color="accent"
+            onClick={() => {
+              core.application.navigateToApp('management', {
+                path: `/kibana/settings?query=${useFieldExistenceSamplingKey}`,
+              });
+            }}
+          >
+            <FormattedMessage
+              id="xpack.lens.indexPattern.useFieldExistenceSamplingLink"
+              defaultMessage="Use field existence sampling"
+            />
+          </EuiLink>
+        );
+
+        deprecatedMessages.push(
+          <EuiCallOut
+            color="warning"
+            iconType="alert"
+            size="s"
+            title={
+              <FormattedMessage
+                id="xpack.lens.indexPattern.useFieldExistenceSamplingBody"
+                defaultMessage="Use field existence sampling is deprecated and will be removed in 8.6.0. To disable, go to {advancedSettings} > {link}"
+                values={{
+                  advancedSettings: (
+                    <strong>
+                      <FormattedMessage
+                        id="xpack.lens.indexPattern.useFieldExistenceSampling.advancedSettings"
+                        defaultMessage="Advanced Settings"
+                      />
+                    </strong>
+                  ),
+                  link,
+                }}
+              />
```

Comment:
Any thoughts on slightly rewording this message to the following and removing any current `strong`/bolding: `Field existence sampling has been deprecated and will be removed in Kibana 8.6.0. You may disable this feature in [advanced settings](link-to-setting).`

CCing @KOTungseth in case she disagrees.

Reply:
Updated: 
<img width="1224" alt="image" src="https://user-images.githubusercontent.com/20072247/188846191-6221559b-8d05-46b3-ad9b-e727e4afd0ba.png">